simpleclient_servlet/src/main/java/io/prometheus/client/filter/MetricsFilter.java

```diff
@@ -171,6 +177,8 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         HttpServletRequest request = (HttpServletRequest) servletRequest;
 
         String path = request.getRequestURI();
+        if (stripContextPath)
```

Comment:
We have `{` `}` in one line `if` statements everywhere else in the code, so please add `{` `}` here as well.

Reply:
Should be fine now.